pcdsdevices/smarpod.py

```diff
@@ -0,0 +1,133 @@
+    cmd_rx = Cpt(EpicsSignal, ':CMD:RX', kind='normal')
+    cmd_ry = Cpt(EpicsSignal, ':CMD:RY', kind='normal')
+    cmd_rz = Cpt(EpicsSignal, ':CMD:RZ', kind='normal')
+    # Stop
+    cmd_stop = Cpt(EpicsSignal, ':CMD:STOP', kind='normal', doc='Stops SmarPod Movements')
+    # Pose reachable?
+    cmd_reachable = Cpt(EpicsSignal, ':CMD:REACHABLE', kind='normal')
+    reachable = Cpt(EpicsSignalRO, ':REACHABLE', kind='normal', doc='Test if the SP can reach a pose. Does not move the SP')
+    # Positions - Readback
+    cmd_pos_rbv = Cpt(EpicsSignal, ':CMD:POS_RBV', kind='normal', doc='Readback current pose position')
+    x_m = Cpt(EpicsSignalRO, ':X_M', kind='normal')
+    y_m = Cpt(EpicsSignalRO, ':Y_M', kind='normal')
+    z_m = Cpt(EpicsSignalRO, ':Z_M', kind='normal')
+    x = Cpt(EpicsSignalRO, ':X', kind='normal')
+    y = Cpt(EpicsSignalRO, ':Y', kind='normal')
+    z = Cpt(EpicsSignalRO, ':Z', kind='normal')
+    rx = Cpt(EpicsSignalRO, ':RX', kind='normal')
+    ry = Cpt(EpicsSignalRO, ':RY', kind='normal')
+    rz = Cpt(EpicsSignalRO, ':RZ', kind='normal')
+    # Movement - Readback
+    moving = Cpt(EpicsSignal, ':MOVING', kind='normal', doc='Movement status')
+    # Movement - Sync
+    cmd_sync = Cpt(EpicsSignal, ':CMD:SYNC', kind='normal')
+    error_code = Cpt(EpicsSignalRO, ':ERROR_CODE', kind='normal')
+    error_desc = Cpt(EpicsSignalRO, ':ERROR_DESC', kind='normal')
+    # Pivot-point
+    cmd_pivot_rbv = Cpt(EpicsSignal, ':CMD:PIVOT_RBV', kind='normal', doc='Readback pivot point')
+    cmd_set_pivot = Cpt(EpicsSignal, ':CMD:SET_PIVOT', kind='normal')
+    cmd_sync_pivot = Cpt(EpicsSignal, ':CMD:SYNC_PIVOT', kind='normal')
+    cmd_pivot_mode = Cpt(EpicsSignal, ':CMD:PIVOT_MODE', kind='normal')
+    pivot_mode = Cpt(EpicsSignal, ':PIVOT_MODE', kind='normal', doc='Pivot mode')
+    # Pivot positions
+    px_m = Cpt(EpicsSignal, ':PX_M', kind='normal')
+    py_m = Cpt(EpicsSignal, ':PY_M', kind='normal')
+    pz_m = Cpt(EpicsSignal, ':PZ_M', kind='normal')
+    px = Cpt(EpicsSignal, ':PX', kind='normal')
+    py = Cpt(EpicsSignal, ':PY', kind='normal')
+    pz = Cpt(EpicsSignal, ':PZ', kind='normal')
+    # Pivot request positions
+    cmd_px = Cpt(EpicsSignal, ':CMD:PX', kind='normal')
+    cmd_py = Cpt(EpicsSignal, ':CMD:PY', kind='normal')
+    cmd_pz = Cpt(EpicsSignal, ':CMD:PZ', kind='normal')
+    # Miscellaneous
+    cmd_calibrate = Cpt(EpicsSignal, ':CMD:CALIBRATE', kind='normal')
+    cmd_read_ver = Cpt(EpicsSignal, ':CMD:READ_VER', kind='normal')
+    ver_sys = Cpt(EpicsSignalRO, ':VER:SYS', kind='omitted')
+    # Device Info
+    ver_sn = Cpt(EpicsSignalRO, ':VER:SN', kind='omitted')
+    ver_product = Cpt(EpicsSignalRO, ':VER:PRODUCT', kind='omitted')
+    ver_firmware = Cpt(EpicsSignalRO, ':VER:FIRMWARE', kind='omitted')
+    # Status
+    cmd_read_status = Cpt(EpicsSignal, ':CMD:READ_STATUS', kind='normal')
+    status_temp = Cpt(EpicsSignalRO, ':STATUS:TEMP', kind='omitted')
```

Comment:
All of this is still really jumbled together. Consider that it might make more sense to shove status-related things into a SmarPodStatus device, which could then be instantiated with a `Cpt(SmarPodStatus, ":STATUS")` much like how I was talking about the poses.